db/schema.rb

```diff
@@ -0,0 +1,69 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 20_231_128_190_630) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.bigint 'user_id', null: false
+    t.bigint 'post_id', null: false
+    t.text 'text'
+    t.datetime 'update_at'
+    t.datetime 'create_at'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+    t.index ['user_id'], name: 'index_comments_on_user_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.bigint 'user_id', null: false
+    t.bigint 'post_id', null: false
+    t.datetime 'update_at'
+    t.datetime 'create_at'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+    t.index ['user_id'], name: 'index_likes_on_user_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.bigint 'author_id', null: false
+    t.string 'title'
+    t.text 'text'
+    t.datetime 'update_at'
+    t.datetime 'create_at'
+    t.integer 'commentsCounter'
+    t.integer 'likesCounter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.text 'bio'
+    t.datetime 'update_at'
+    t.datetime 'create_at'
+    t.integer 'postsCounter'
```

Comment:
- [ ] [Optional] I would suggest to use `Snake Case` for column names instead of `Camel Case`. It's advisable to avoid single-letter or exceedingly brief names for column names that lack clarity.
  - Example: `posts_counter`, `comments_counter`, `likes_counter`.